scripts/apps/authoring/authoring/ValidateCharacters.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ng from 'core/services/ng';
+import {gettext} from 'core/utils';
+
+interface IProps {
+    item: string;
+    field: string;
+    onChange: (field: string, isError: boolean) => void;
+}
+
+export class ValidateCharacters extends React.PureComponent<IProps> {
+    render() {
+        const {item, field, onChange} = this.props;
+        const config = ng.get('$rootScope').config;
+        let invalidCharError = null;
+
+        if (item == null || field == null) {
+            return null;
+        }
+
+        if (config != null && config.disallowedCharacters != null) {
```

Comment:
Hi @tomaskikutis disallowed characters for a field are global, that's why we wanted to have them in one place to avoid the need to update all content profiles when you decide to update the list of characters.

Reply:
:+1: let's keep it in the config then